package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@knocklabs/node",
-  "version": "0.4.13",
+  "version": "0.4.14",
```

Comment:
Nit: prefer doing this in a separate commit after this is merged

Reply:
Will remove!